reactor-core/src/test/java/reactor/core/ScannableTest.java

```diff
@@ -251,7 +251,7 @@ public void taggedAppendedFluxTest() {
 				    .tag("2", "Two");
 
 		final Stream<Tuple2<String, String>> scannedTags = Scannable.from(tagged1).tags();
-		assertThat(scannedTags.iterator())
+		assertThat(scannedTags)
```

Comment:
Just wondering why there we use stream without `.iterator()).toIterable()` but in the above changes we do with

Reply:
looks like there's no particular reason not to directly assert the `Stream` itself since AssertJ now supports all the necessary `containsXxx` assertions on these, will change.